django/db/backends/sqlite3/introspection.py

```diff
@@ -67,6 +67,32 @@ def get_field_type(self, data_type, description):
             return 'JSONField'
         return field_type
 
+    def only_nonempty_tables(self, tables):
```

Comment:
Unless merging into `.django_table_names()` with a `skip_empty` argument:
```suggestion
    def exclude_empty_tables(self, tables):
```